.github/workflows/ci.yml

```diff
@@ -23,5 +23,49 @@ jobs:
           pip install .
       - name: Run tests
         run: |
-          pip install pytest==6.1.2
+          pip install pytest==6.2.5
           pytest
+
+  # This job checks whether the current workflow is triggered by a new tag with the following form: INT.INT or INT.INT.INT (for example: 1.0, 1.0.1, ...)
+  # The result is saved in a variable and used as a conditional variable when uploading packages.
+  check-release-tag:
+    name: "Check for new release"
+    runs-on: ubuntu-latest
+    outputs:
+      is-release: ${{ steps.do-check.outputs.is-release }}
+    steps:
+      - name: Check release tag ${{ github.ref }}
+        id: do-check
+        run: |
+          if [[ ${{ github.ref }} =~ ^refs\/tags\/[0-9]+\.[0-9]+(\.[0-9]+)?$ ]]; then
```

Comment:
It's too late to change this for NetworKit, but IMHO adding a `v` in front of the version (for example, v1.0) is a bit nicer.

Reply:
Good point, I thought the same when we introduced the automatic release for NetworKit.